gcloud/README.md

```diff
@@ -1,18 +1,25 @@
-# gcloud
+# Tool builder: `gcr.io/cloud-builders/gcloud`
 
-This is a tool builder to simply invoke
-[`gcloud`](https://cloud.google.com/sdk/gcloud/) commands.
-
-Arguments passed to this builder will be passed to `gcloud` directly, allowing
-callers to run [any `gcloud`
-command](https://cloud.google.com/sdk/gcloud/reference/).
+This Cloud Build builder runs the
+[`gcloud`](https://cloud.google.com/sdk/gcloud/) tool to interact with Google
+Cloud Platform resources.
 
 When executed in the Cloud Build environment, commands are executed with
 credentials of the [builder service
 account](https://cloud.google.com/cloud-build/docs/permissions) for the
-project.
+project. The latest released version of `gcloud` is used.
+
+You should consider instead using an [official `google/cloud-sdk`
+image](https://hub.docker.com/r/google/cloud-sdk) and specifying the `gcloud` entrypoint:
+
+```yaml
+steps:
```

Comment:
Consider adding an example .yaml that explains "these two build steps are functionally equivalent" to make the difference between our image (with entrypoint) and the FROM image (without) explicit.